src/Functions/SubtractSubSeconds.cpp

```diff
@@ -1,28 +0,0 @@
-#include <Functions/FunctionFactory.h>
-#include <Functions/FunctionDateOrDateTimeAddInterval.h>
-
-
-namespace DB
-{
-
-using FunctionSubtractNanoseconds = FunctionDateOrDateTimeAddInterval<SubtractNanosecondsImpl>;
-REGISTER_FUNCTION(SubtractNanoseconds)
```

Comment:
why was it split?

Reply:
Just for consistency with existing files subtractDays.cpp, subtractHours.cpp, subtractMinutes.cpp, subtractSeconds.cpp etc. The split is part of a small refactoring unrelated to the actual changes (see the PR message).